services/movie.py

```diff
@@ -0,0 +1,44 @@
+from typing import List
+
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        actors_ids: List[int] = None,
+        genres_ids: List[int] = None
+) -> QuerySet:
```

Comment:
You could indicate that you return a queryset of `Movie` instances in the type annotation.